File: 16challenge/main.r
cigarettes_per_day <- as.numeric(readline("Number of cigarettes smoked per day: "))
years_smoking <- as.numeric(readline("Number of years smoking: "))
minutes_lost_per_cigarette <- 10
minutes_lost_per_day <- cigarettes_per_day * minutes_lost_per_cigarette
days_lost_per_year <- minutes_lost_per_day / (24 * 60)
total_days_lost <- days_lost_per_year * years_smoking
print(paste0("The smoker will lose a total of ", total_days_lost, " days of life."))

File: 16challenge/main.java
import java.util.Scanner;

public class LifeExpectancy {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Number of cigarettes smoked per day: ");
        int cigarettesPerDay = scanner.nextInt();
        System.out.print("Number of years smoking: ");
        int yearsSmoking = scanner.nextInt();
        scanner.close();

        int minutesLostPerCigarette = 10;
        int minutesLostPerDay = cigarettesPerDay * minutesLostPerCigarette;
        double daysLostPerYear = (double) minutesLostPerDay / (24 * 60);
        double totalDaysLost = daysLostPerYear * yearsSmoking;
        System.out.println("The smoker will lose a total of " + totalDaysLost + " days of life.");
    }
}
